back-end/tests/models/users.test.js

```diff
@@ -56,31 +56,31 @@ describe("user model", () => {
       const questIds = adventurerQuests.map((q) => q.id).sort();
       expect(questIds).toEqual([quest1.id, quest2.id].sort());
     });
+  });
```

Comment:
By the way, it's possible to have a `describe()` inside a `describe()`. Because of that change of indentation here, the `adventurer` variable below (currently line 68) is now longer defined, since it was defined in the `beforeEach()`.